std_msgs/Header header

sensor_msgs/PointCloud2 cluster

autoware_auto_perception_msgs/DetectedObject detected_object